src/ComputeSharp.D2D1/Shaders/Interop/Effects/PixelShaderEffect.cs

```diff
@@ -379,6 +355,23 @@ private uint Release()
                 _ = this.d2D1DrawInfo->Release();
```

Comment:
it's a good idea to set these to `null` when they're released

Reply:
same goes for the other pointers being freed or released in this method

Reply:
This method is interlocked and also it's releasing the whole instance right after releasing these COM objects anyway, so I don't think it's needed to set them to `null`. The whole object is being destroyed anyway 😄